pkg/mcs/scheduling/server/cluster.go

```diff
@@ -612,26 +612,30 @@ func (c *Cluster) processRegionHeartbeat(ctx *core.MetaProcessContext, region *c
 		// Due to some config changes need to update the region stats as well,
 		// so we do some extra checks here.
 		if hasRegionStats && c.regionStats.RegionStatsNeedUpdate(region) {
-			ctx.TaskRunner.RunTask(
+			if err := ctx.TaskRunner.RunTask(
 				ctx,
 				ratelimit.ObserveRegionStatsAsync,
 				func(_ context.Context) {
 					if c.regionStats.RegionStatsNeedUpdate(region) {
 						cluster.Collect(c, region, hasRegionStats)
 					}
 				},
-			)
+			); err != nil {
+				log.Warn("run task failed", zap.String("name", ratelimit.ObserveRegionStatsAsync), zap.Error(err))
```

Comment:
I prefer not to print the log here since it is in the hot path.

Reply:
If that's the case, how about not returning an error in `RunTask`?

Reply:
ping @rleungx

Reply:
Use metrics or ignore them are both ok to me